### DISCUSSION TITLE
Zenbook S16 _ How to avoid laggy behaviour when unplugged

### DISCUSSION THREAD

**Discussion by sunlooksgreen** [seerge/g-helper#3258] [General]
Hello,

Just got myself a Zenbook S16 with the ryzen 370 processor and thanks to this little app I got to keep the laptop cool enought when plugged in.. Congrats to the developer. Now there's another problem I am facing and already tried many things but nothing worked so far.. When I am unplugged the processor base speed drops from 2.5 to 1.5GHz in order to save battery (as pointed many times on this forum) The thing is that coming from Intel I do find the lower base speed too agressive and simple things like contextual menus become laggy, too laggy for my taste and even dealbreaker to the point of **wanting to return the laptop**. Does anyone know if there is a way to force the processor to stay "awake" just the same as when plugged? I have tried all power modes, even the full performance one, to no avail.. 

Thanks in advance
Albert

**Comment by seerge** [comment]
@sunlooksgreen hello, I don't know :) but if it gets slow on battery - may be it's something wrong with power supply in that case ? (i.e. battery not providing enough power for device to work correctly) ? Did you try changing different modes on battery ?

**Comment by sunlooksgreen** [comment]
Yes I did try all performance modes; and when I launch a render it goes to the tdp's that it is supposed to go. It's just that the basic state is 1 full GHz slower than plugged in and you do notice that a lot, specially in cad sofware and general navigation around folders.. I read a test over the previous generation, specifically the 7840u, that the reviewer said the performance drop when on battery was too much compared to the intel counterpart. This seemed weird to me at the time but now I understand why he said so.. It would seem as if Amd is beeing more agressive with the clock down to improve battery times..

**Comment by sunlooksgreen** [comment]
you have this video on reddit on a similar laptop that shows the issue; seems others have noticed as well..

https://www.reddit.com/r/ASUS/comments/1dqry6i/asus_vivobook_16_cpu_runs_extremely_slow_on/

and throttlestop, the solution suggested, doesn't support strix point so far

**Reply by seerge** [reply]
@sunlooksgreen then it looks like some firmware issue. Did you try updating to the latest BIOS ? Throttlestop is Intel only as far as I remember :)

**Reply by sunlooksgreen** [reply]
mm.. doesn't seem to me like a firmware issue; I think, as I said before, it is designed to work that way. The thing is some people don't care and others do..  And I thought maybe someone here could lend a hand..

PD: I updated to latest firmware, but nothing changed :(

**Comment by robscheele** [comment]
Hello Seerge, i am facing exactly the same problem as you describe. I have updated all the drivers and bios, and tweaked all the power settings to avoid the problem without any luck. Please let me know if you have found a solution and i will do the same. Otherwise i will indeed also return the laptop.

**Comment by robscheele** [comment]
OK this could be another solution 
![image](https://github.com/user-attachments/assets/330cce45-ed5c-4bcb-91f7-f9440667f630)

**Reply by seerge** [reply]
@robscheele of setting minimum processor status to 100% helps - this is most likely some kind of mistuning on firmware level (so when it's below that CPU does not get enough power at all).

Does setting this to something like 50% also help ?

**Reply by robscheele** [reply]
I would set it to 100% so that the CPU operates at full capacity, regardless of the laptop's battery status. I'm currently draining the battery to observe how it behaves once it runs out and is plugged back in. Please let me know if this solution works for you as well.

**Reply by seerge** [reply]
@robscheele well, i don't have Zenbook S16 - so i'm not really facing this problem in a first place.

**Comment by robscheele** [comment]
oke, i see, sunlooksgreen should reply

**Comment by sunlooksgreen** [comment]
Hello guys, thanks a lot for your replies!! I will give it a try later on, once I have access to the computer, and report back..

**Comment by robscheele** [comment]
Disable in bios the option advanced energy management, fixes all problems

Op zo 20 okt 2024 om 14:26 schreef sunlooksgreen ***@***.***>

**Reply by sunlooksgreen** [reply]
Hello Robscheele, 

you were spot on.. just tried disabling advanced energy management and it worked.. now the base processor frequency follows the energy mode and the computer works as it should.. It's called **ERP on the bios** 

copy paste from google:

_What is ERP in ASUS BIOS?
ERP (Energy-Related Products) mode in the BIOS (Basic Input/Output System) is a power management feature that focuses on energy efficiency and reducing power consumption in computer systems. By enabling ERP mode, you can optimize your system's power usage during periods of inactivity or when the system is shutdown._


Thanks for your answers, hope this helps someone else as well
